randovania/games/samus_returns/logic_database/Area 2 Dam Exterior.txt

```diff
@@ -482,8 +482,16 @@ Extra - asset_id: collision_camera_024
   * Tunnel to Metroid Caverns Entrance/Tunnel to Spike Ravine (Bottom)
   > Pickup (Aeion Tank)
       Any of the following:
+          # IBJ from below
           Simple IBJ
-          Morph Ball and Space Jump and Movement (Intermediate)
+          All of the following:
+              # Dodge the spikes
+              Morph Ball
+              Any of the following:
+                  # With Space Jump
+                  Space Jump and Movement (Intermediate)
+                  # With Super Jump and Unmorph Extend https://youtu.be/UxgWgzcCDxk
+                  High Jump Boots and Super Jump (Advanced) and Unmorph Extend (Advanced)
```

Comment:
I assume the difficulty is based on you trying it?

Reply:
No I haven't tried this. Advanced was the suggested difficulty.